ksql-engine/src/main/java/io/confluent/ksql/function/udf/math/Round.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.function.udf.math;
+
+import io.confluent.ksql.function.udf.Udf;
+import io.confluent.ksql.function.udf.UdfDescription;
+import io.confluent.ksql.function.udf.UdfParameter;
+import io.confluent.ksql.function.udf.UdfSchemaProvider;
+import io.confluent.ksql.schema.ksql.SqlBaseType;
+import io.confluent.ksql.schema.ksql.types.SqlDecimal;
+import io.confluent.ksql.schema.ksql.types.SqlType;
+import io.confluent.ksql.util.KsqlException;
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.sql.SQLDataException;
+import java.util.List;
+
+/*
+The rounding behaviour implemented here follows that of java.lang.Math.round()
```

Comment:
Would this comment not be better as a javadoc? i.e. starting with `/**`?

Reply:
JavaDoc is more for user facing API docs. In this case the comment is aimed at the ksql developer so I don't think making it JavaDoc adds anything here